examples/Max_zoom_tests.ipynb

```diff
@@ -0,0 +1,113 @@
+{
```

Comment:
Suggestion for the notebook name: `Max_zoom.ipynb`. Also can you clean the cell outputs?